zebra-rpc/src/server.rs

```diff
@@ -84,7 +84,7 @@ impl RpcServer {
         mempool: Buffer<Mempool, mempool::Request>,
         state: State,
         #[cfg_attr(not(feature = "getblocktemplate-rpcs"), allow(unused_variables))]
-        chain_verifier: ChainVerifier,
+        router_verifier: ChainVerifier,
```

Comment:
This generic probably needs to be renamed, this can go in the next ticket

```suggestion
        router_verifier: BlockVerifierRouter,
```

Reply:
This generic rename was done in this pull request at https://github.com/ZcashFoundation/zebra/pull/6762/commits/d415d44212a92cf8f9c28a1c7f6196c73cd99d75